python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -33,7 +35,7 @@ class LoadFileOperator(AstroSQLBaseOperator):
     :param native_support_kwargs: kwargs to be used by method involved in native support flow
     :param columns_names_capitalization: determines whether to convert all columns to lowercase/uppercase
             in the resulting dataframe
-    :param enable_native_fallback: Use enable_native_fallback=True to fall back to default transfer
+    :param enable_native_fallback: (deprecated) Use enable_native_fallback=True to fall back to default transfer
```

Comment:
I would say instead of deprecating, we allow both configs (global and local)

So users can disable it globally but selectively enable it based on a certain transfer

Reply:
@kaxil I believe global config means airflow config here but can you please elaborate a bit on what you mean by a local config (the existing one i.e based on the value of the `enable_native_fallback ` param)? Also when you say
> selectively enable it based on a certain transfer

do you mean using `enable_native_fallback` in the operator?

Reply:
Removed deprecation, now we have global and local config. the local config will overwrite the global one